source/applications/basic/visualization/read_zdf_vis_3d.py

```diff
@@ -50,51 +49,37 @@ def _display_depthmap(xyz):
     plt.show(block=False)
 
 
-def _get_mid_point(xyz):
-    """Calculate mid point from average of the 100 centermost points.
-
-    Args:
-        xyz: X, Y and Z images (point cloud co-ordinates)
-
-    Returns:
-        mid_point: Calculated mid point
-
-    """
-    xyz_center_cube = xyz[
-        int(xyz.shape[0] / 2 - 5) : int(xyz.shape[0] / 2 + 5),
-        int(xyz.shape[1] / 2 - 5) : int(xyz.shape[1] / 2 + 5),
-        :,
-    ]
-    return (
-        np.nanmedian(xyz_center_cube[:, :, 0]),
-        np.nanmedian(xyz_center_cube[:, :, 1]),
-        np.nanmedian(xyz_center_cube[:, :, 2]),
-    )
-
-
 def _display_pointcloud(rgb, xyz):
     """Display point cloud.
 
     Display the provided point cloud `xyz`, and color it with `rgb`.
 
-    We take the centermost co-ordinate as 'lookat' point. We assume that
-    camera location is at azimuth -pi/2 and elevation -pi/2 relative to
-    the 'lookat' point.
-
     Args:
         rgb: RGB image
         xyz: X, Y and Z images (point cloud co-ordinates)
 
     Returns None
 
     """
-    mid_point = _get_mid_point(xyz)
-    point_cloud_to_view = xyz
-    point_cloud_to_view[np.isnan(xyz[:, :, 2])] = 0
-    viewer = pptk.viewer(point_cloud_to_view)
-    viewer.attributes(rgb.reshape(-1, 3) / 255.0)
-    viewer.set(lookat=mid_point)
-    viewer.set(phi=-math.pi / 2, theta=-math.pi / 2, r=mid_point[2])
+    point_cloud_to_view = np.nan_to_num(xyz)
```

Comment:
Does this create fake points in the point cloud?

Reply:
Why 'point_cloud_to_view'?

Reply:
Do we need to remove the nan points? or can open3d handle the nans?

Reply:
We need to remove them otherwise we can't see a point cloud

Reply:
Then we need to align on what we want to do. By default it will replace nans to zeros. the downside is that if you zoom to (0,0,0) you will probably see a point (many points stacked one on top of each other). Alternative is to set nans to a value we like (it could be a very large positive or negative number). I am not saying this is better, since it has the same downside (if you zoom in there you will see the point(s)). What is your opinion on this @chrisasc @robertkt1? Anything I missed? I guess nan to zero is common and ok to do?

Reference: https://numpy.org/doc/stable/reference/generated/numpy.nan_to_num.html

Reply:
One possibility is to remove all NaN points, but this would make the point cloud unorganized. I think that converting them to (0,0,0) is the best option as this is what's common to do (it happens in Halcon, we do it in C++, and I believe that's what most open-source libraries do). 

Unless @robertkt1 has a good argument for not converting to (0,0,0), I will keep doing this.